tests/foreman/cli/test_errata.py

```diff
@@ -1390,9 +1448,116 @@ def test_positive_user_permission(self):
+        self.assertEqual(user['admin'], 'no')
+        self.assertEqual(set(user['roles']), {DEFAULT_ROLE, role['name']})
+        # try to get organization info
+        # get the info as admin user first
+        org_info = Org.info({'id': org['id']})
+        self.assertEqual(org['id'], org_info['id'])
+        self.assertEqual(org['name'], org_info['name'])
+        # get the organization info as the created user
+        with self.assertRaises(CLIReturnCodeError) as context:
+            Org.with_user(user_name, user_password).info({'id': org['id']})
+        self.assertIn(
+            'Forbidden - server refused to process the request',
+            context.exception.stderr
+        )
+        # try to get the erratum products list as the created user
+        # by product ids
+        product1_erratum = Erratum.with_user(user_name, user_password).list({
+            'organization-id': org['id'],
+            'product-id': product1['id']
+        })
+        self.assertEqual(len(product1_erratum), 4)
+        product1_errata_ids = [
+            errata['errata-id']
+            for errata in product1_erratum
+        ]
+        self.assertIn(FAKE_0_ERRATA_ID, product1_errata_ids)
+        with self.assertRaises(CLIReturnCodeError) as context:
+            Erratum.with_user(user_name, user_password).list({
+                'organization-id': org['id'],
+                'product-id': product2['id']
+            })
+        self.assertIn('repository not found', context.exception.message)
+        # try to get the erratum products list as the created user
+        # by product names
+        product1_erratum = Erratum.with_user(user_name, user_password).list({
+            'organization-id': org['id'],
+            'product': product1['name']
+        })
+        self.assertEqual(len(product1_erratum), 4)
+        product1_errata_ids = [
+            errata['errata-id']
+            for errata in product1_erratum
+        ]
+        self.assertIn(FAKE_0_ERRATA_ID, product1_errata_ids)
+        with self.assertRaises(CLIReturnCodeError) as context:
+            Erratum.with_user(user_name, user_password).list({
+                'organization-id': org['id'],
+                'product': product2['name']
+            })
+        self.assertIn('product not found', context.exception.message)
+        # try to get the erratum products list by organization id only
+        if not bz_bug_is_open('1403947'):
+            # ensure that all products erratum are accessible by admin user
+            admin_org_erratum_info_list = Erratum.list({
+                'organization-id': org['id']})
+            admin_org_errata_ids = [
+                errata['errata-id']
+                for errata in admin_org_erratum_info_list
+            ]
+            self.assertIn(FAKE_0_ERRATA_ID, admin_org_errata_ids)
+            self.assertIn(FAKE_3_ERRATA_ID, admin_org_errata_ids)
+            # product 1 has 4 erratum, product 2 has 73 erratum,
```

Comment:
I think yes, perhaps we should add these numbers as constants as Andrei, so if some one think to change the repo he have to change that numbers too.